x-pack/legacy/plugins/infra/public/pages/logs/log_entry_categories/sections/top_categories/top_categories_table.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiBasicTable, EuiBasicTableColumn } from '@elastic/eui';
+import numeral from '@elastic/numeral';
+import { i18n } from '@kbn/i18n';
+import React, { useMemo } from 'react';
+
+import euiStyled from '../../../../../../../../common/eui_styled_components';
+import {
+  LogEntryCategory,
+  LogEntryCategoryHistogram,
+} from '../../../../../../common/http_api/log_analysis';
+import { TimeRange } from '../../../../../../common/http_api/shared';
+import { AnomalySeverityIndicator } from './anomaly_severity_indicator';
+import { RegularExpressionRepresentation } from './category_expression';
+import { DatasetsList } from './datasets_list';
+import { LogEntryCountSparkline } from './log_entry_count_sparkline';
+
+export const TopCategoriesTable = euiStyled(
+  ({
+    className,
+    timeRange,
+    topCategories,
+  }: {
+    className?: string;
+    timeRange: TimeRange;
+    topCategories: LogEntryCategory[];
+  }) => {
+    const columns = useMemo(() => createColumns(timeRange), [timeRange]);
+
+    return (
+      <EuiBasicTable
+        columns={columns}
+        items={topCategories}
+        rowProps={{ className: `${className} euiTableRow--topAligned` }}
+      />
+    );
+  }
+)`
+  &.euiTableRow--topAligned .euiTableRowCell {
+    vertical-align: top;
+  }
+`;
+
+const createColumns = (timeRange: TimeRange): Array<EuiBasicTableColumn<LogEntryCategory>> => [
+  {
+    align: 'right',
+    field: 'logEntryCount',
+    name: i18n.translate('xpack.infra.logs.logEntryCategories.countColumnTitle', {
+      defaultMessage: 'Message count',
+    }),
+    render: (logEntryCount: number) => {
+      return numeral(logEntryCount).format('0,0');
+    },
+    width: '120px',
```

Comment:
```suggestion
    width: '108px',
```

Reply:
It seems that these widths are highly arbitrary - why do we want to use 108 over 120 px? :thinking: